target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -185,6 +186,11 @@ ramips_setup_macs()
 		wan_mac=$lan_mac
 		label_mac=$lan_mac
 		;;
+	mercusys,mr70x)
+		lan_mac=$(mtd_get_mac_binary factory 0x8)
+		wan_mac=$(macaddr_add "$lan_mac" 1)
+		label_mac=$lan_mac
+		;;
```

Comment:
In the first revision you had the MAC setup in the dts, why did you move it here?

Reply:
I would also prefer if the mac addresses are configured in the dts file and not in this script where possible.